docs/node-developers/sandbox-node.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: "Sandbox Node"
+---
+
+# Sandbox Node
+
+The Mina Sandbox Node will enable you to test and get familiar with core features of the protocol and build tooling in a stable environment -- it's a single-node private network that uses the same configuration as the live testnet. This sandbox supports multiple accounts, sending transactions between them, and will also support performing SNARK work, delegating, and staking. In fact since it's a single node network, you'll get all the block rewards!
+
+:::info
+
+The sandbox does **NOT** connect you to a live network.
+
+:::
+
+## Installation
+
+[Docker](https://www.docker.com) is a tool for portably running applications. The Mina Sandbox is packaged with Docker, and now built-in to our daemon containers. It’s easy to install--we suggest the [Docker Desktop](https://www.docker.com/products/docker-desktop).
+
+After you have Docker installed run the following command to spin up the Mina Sandbox.
+
+```
+docker run \
+  --publish 3085:3085 \
+  -d \
+  --name mina \
+  -e RUN_DEMO=true \
+  -e MINA_PRIVKEY_PASS='' \
+  minaprotocol/mina-daemon:1.3.0-9b0369c-bullseye-devnet
+
+```
+
+This command will start a daemon inside the docker container and expose the GraphQL port (3085), which is used for communication with the client, to your computer. This daemon will automatically be running in the background with a block producer and snark worker.
+
+You can view logs by executing.
+
+```
+docker logs --follow mina
+```
+
+And stop mina by running.
+
+```
+docker stop mina
+```
+
+You can use the Mina CLI to interact with the sandbox node. The following command opens a shell inside the docker container from where you can issue any of the available [mina commands](/node-operators/cli-reference).
```

Comment:
Can we change this link text to `cli commands` please? I know it's old content, but it'll make more sense to readers.

Reply:
Done